EVENT.md

```diff
@@ -1129,6 +1147,10 @@ the [denial of service](#denial-of-service) for more details. To prevent this
 attack, the browser should maintain these limits per reporting site. This
 effectively limits the number of unique sites covered per {source site, reporting site} applied to all unexpired sources regardless of type at source time.
 
+The browser can also limit the number of `destination` sites per {source site, reporting site, 1 day}
+to mitigate the history reconstruction attack as new `destination` sites can be
+registered when the `priority_fifo` algorithm is used.
```

Comment:
This mention of an algorithm is obsolete IIUC.

Reply:
Removed, thanks.